generator/src/test/java/com/graphicsfuzz/generator/transformation/DonateLiveCodeTransformationTest.java

```diff
@@ -407,7 +409,10 @@ public void testArrayAccessesAreInBounds() throws Exception {
           GenerationParams.normal(ShaderKind.FRAGMENT, true)
       );
 
```

Comment:
Can you add a comment here to explain the logic around failCount?  I guess you must have found that when you upped the seed limit, there were some cases where live code injection was failing (i.e., no code was injected), and this just checks that it doesn't fail too often?  It would be worth remarking that it is theoretically possible for this test to fail due to bad luck.

Reply:
Done